uberfire-commons/src/main/java/org/uberfire/commons/concurrent/ExecutorServiceProducer.java

```diff
@@ -60,9 +63,15 @@ protected ExecutorService buildFixedThreadPoolExecutorService(String key, int de
         String stringProperty = System.getProperty(key);
         int threadLimit = stringProperty == null ? defaultLimit : toInteger(stringProperty);
         if (threadLimit > 0) {
+            if (logger.isDebugEnabled()) {
```

Comment:
`if (logger.isDebugEnabled()) {`
Is this used because of performance reasons?

Reply:
That's right. Is not necessary to log things if we are not in debug mode